### PR TITLE
Added a way for oom events to be handled by the manager

### PR DESCRIPTION
**Pull request by kateknister**

### CODE REVIEW

**Comment by vmarmol** [comment]
Looks like it needs a rebase :)

**Comment by vmarmol** [comment]
ok to test

**Comment by kateknister** [comment]
Does it still need a rebase?

File: manager/manager.go [hunk 1]
```diff
@@ -645,7 +655,21 @@ func (self *manager) watchForNewOoms() error {
...
+			}
 		}
+		return nil
 	}()
 	return nil
 }
+
+// can be called by the api which will take events returned on the channel
+func (self *manager) WatchForEvents(request *events.Request, passedChannel chan *events.Event) error {
+	err := self.eventHandler.WatchEvents(passedChannel, request)
```
Comment (vmarmol) [inline manager/manager.go]: We can just return the result here

File: manager/manager.go [hunk 2]
```diff
@@ -645,7 +655,21 @@ func (self *manager) watchForNewOoms() error {
 				EventData:     oomInstance,
 			}
 			glog.V(1).Infof("Created an oom event: %v", newEvent)
+			err := self.eventHandler.AddEvent(newEvent)
```
Comment (vmarmol) [inline manager/manager.go]: Glog and error instead. We probably want this to run forever

**Comment by vmarmol** [comment]
Yep, shows one commit now :)

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**